cugraph-analytics/cugraph_analytics.ipynb

```diff
@@ -0,0 +1,453 @@
+    "\n",
+    "Everything inside `MAGE` is integrated in a way to make executing PageRank really easy. By running the following line, you will get results in ~4 seconds for our huge graph. The following query will first execute the algorithm, and then create and set the `rank` property\n",
+    "of each node to the value that the `cugraph.pagerank` algorithm returned. The value of that property will then be saved as a variable `rank`. This whole query will be executed in ~4 seconds for our graph of more than 1 million edges.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "  memgraph.execute(\n",
+    "        \"\"\"\n",
+    "        CALL cugraph.pagerank.get() YIELD node,rank\n",
+    "        SET node.rank = rank;\n",
+    "        \"\"\"\n",
+    "    )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, ranks are ready and you can retrieve them with the following Python call:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
+    }
+   },
+   "outputs": [],
+   "source": [
+    "results =  memgraph.execute_and_fetch(\n",
+    "        \"\"\"\n",
+    "        MATCH (n)\n",
+    "        RETURN n.id as node, n.rank as rank\n",
+    "        ORDER BY rank DESC\n",
+    "        LIMIT 10;\n",
+    "        \"\"\"\n",
+    "    )\n",
+    "for dict_result in results:\n",
+    "    print(f\"node id: {dict_result['node']}, rank: {dict_result['rank']}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the code above, we returned 10 nodes with the highest rank score. Results are available in a dictionary form. Now it is time to visualize results with **[Memgraph Lab](https://memgraph.com/lab)**. Besides creating beautiful visualizations powered by [D3.js](https://d3js.org/) and our [graph style script](https://memgraph.com/docs/memgraph-lab/graph-style-script-language), you can use **Memgraph Lab** to [query graph database](https://memgraph.com/docs/memgraph-lab/connect-to-memgraph#executing-queries) and write your graph algorithms in **Python or C++ or even Rust**, check Memgraph Database Logs, and visualize graph schema. If you don't have your own dataset on hand, there are plenty of datasets available in Memgraph Lab that you can explore. Everything you might need to know about Memgraph Lab can be found in our [docs](https://memgraph.com/docs/memgraph-lab/).\n",
```

Comment:
```suggestion
    "With the above code, we got the 10 nodes with the highest rank score in a Python dictionary. Time to visualize the results with **[Memgraph Lab](https://memgraph.com/lab)**! Besides creating beautiful visualizations powered by [D3.js](https://d3js.org/) and our [graph style script](https://memgraph.com/docs/memgraph-lab/graph-style-script-language), you can use Memgraph Lab to [query your graph database](https://memgraph.com/docs/memgraph-lab/connect-to-memgraph#executing-queries) and write graph algorithms in **Python**, **C++**, or even **Rust**, check Memgraph Database Logs, and visualize graph schema. If you don't have your own dataset at hand, there are plenty of datasets available in Memgraph Lab that you can explore. Everything you might need to know about Memgraph Lab can be found in our [docs](https://memgraph.com/docs/memgraph-lab/).\n",
```

Reply:
By the way, I’d put the part starting with "Besides creating beautiful visualizations" after the following query and its explanation. Also consider making a bulleted list of all the things Lab can do.